src/Popover.tsx

```diff
@@ -22,7 +22,7 @@ export type PopoverProps = {
   position?: 'top' | 'right' | 'bottom' | 'left';
 } & ViewProps;
 
-export default React.forwardRef<View, PopoverProps>(
+const Popover = React.forwardRef<View, PopoverProps>(
   (
```

Comment:
This should also be a function, not an arrow function, since it won't inherit the name of the outer variable. So it should be `forwardRef(function Popover() {`